client/src/App.vue

```diff
@@ -1090,201 +1091,120 @@
+								<i class="fas fa-exclamation-triangle yellow"></i> This is the exact same website but logs and preferences do not transfert over domains, you can still retrieve your game logs by accessing the <a href="https://mtgadraft.herokuapp.com">old URL</a>.  
+							</p>
 						</div>
-						<div class="welcome-section">
-							Each player can import their MTGA collection to restrict the card pool to cards they own.
-							(Session owners can bypass this feature by enabling "Ignore Collections"):
-							<ol>
-								<li>
-									Enable "Detailed Logs" in MTG Arena. It is required for the collection import to
-									work. The toggle can be found in "Options > Account > Detailed Logs (Plugin
-									Support)".
-								</li>
-								<li>
-									<a onclick="document.querySelector('#file-input').click()">Upload</a>
-									your MTGA log file "Player.log" located in
-									<tt
-										class="clickable"
-										@click="logPathToClipboard"
-										v-tooltip="'Copy path to clipboard'"
-										>C:\Users\%username%\AppData\LocalLow\Wizards Of The Coast\MTGA\</tt
-									>
-									(Note:
-									<a
-										href="https://support.microsoft.com/en-us/help/14201/windows-show-hidden-files"
-										target="_blank" rel="noopener nofollow"
-									>
-										AppData folder is hidden by default
-										<i class="fas fa-external-link-alt"></i>
-									</a>
-									).
-								</li>
-							</ol>
+						<div class="news">
+							<em>March 06, 2021</em>
+							<p>
+								<img src="img/sets/tsr.svg" class="set-icon" style="--invertedness: 100%" />
+								Time Spiral Remastered (TSR) is now available! (see the "<i class="fas fa-ellipsis-h"></i> More sets..." option)<br />
+							</p>
 						</div>
 					</div>
-					<div class="container">
-						<div class="section-title">
-							<h2>Help</h2>
-						</div>
-						<div class="welcome-section">
-							Visit the
-							<a @click="displayedModal = 'help'">FAQ / Help</a>
-							section.
-							<br />For any question/bug report/feature request you can email to
-							<a href="mailto:mtgadraft@gmail.com">mtgadraft@gmail.com</a>
-							or join the
-							<a href="https://discord.gg/XscXXNw">MTGADraft Discord</a>.
-						</div>
+				</div>
+				<div class="container" style="grid-area: Help;">
+					<div class="section-title">
+						<h2>Help</h2>
+					</div>
+					<div class="welcome-section welcome-alt">
+						<a @click="displayedModal = 'gettingStarted'"><i class="fas fa-info-circle"></i> Get Started</a> guide.<br />
+						<br />
+						Visit the <a @click="displayedModal = 'help'">FAQ / Help</a> section.<br />
+						<br />
+						For any question/bug report/feature request you can email to
+						<a href="mailto:mtgadraft@gmail.com">mtgadraft@gmail.com</a>
+						or join the
+						<a href="https://discord.gg/XscXXNw"><i class="fab fa-discord"></i> MTGADraft Discord</a>.
 					</div>
 				</div>
```

Comment:
Played around with the `Help` section.
Tried to archive something like this:
![res](https://user-images.githubusercontent.com/9874850/115029976-d9387080-9ec6-11eb-930f-b858d8450144.png)


```suggestion
				<div class="container" style="grid-area: Help;">
					<div class="section-title">
						<h2>Ressources</h2>
					</div>
					<div class="welcome-section welcome-alt">
						<ul>
							<li>
								<a @click="displayedModal = 'gettingStarted'"><i class="fas fa-rocket"></i> Get Started</a> guide
							</li>
							<li>
								<a @click="displayedModal = 'help'"><i class="fas fa-info-circle"></i> FAQ / Help</a> section
							</li>
						<ul>
						For any questions, bugs or suggestions you can write a
						<a href="mailto:mtgadraft@gmail.com"><i class="fas fa-envelope"></i>mail</a>
						or join the
						<a href="https://discord.gg/XscXXNw"><i class="fab fa-discord"></i> Discord</a>.
					</div>
				</div>
```

Reply:
I like the icons :)
![image](https://user-images.githubusercontent.com/1338143/115052156-e6ad2500-9edd-11eb-9626-c3729063e8e8.png)

I also replaced "options" by "settings" everywhere. "settings" is more appropriate here, right?

Reply:
>I like the icons :)

To the Moon!! 👨‍🚀 

>I also replaced "options" by "settings" everywhere. "settings" is more appropriate here, right?

I like that. The modal title itself still says "Additional Session Options" though. ;)